Documentation/Installation.rst

```diff
@@ -0,0 +1,12 @@
+Installation 
+===========
+
+
+Katydid source can be found on the [Project 8 Github repository](https://github.com/project8/katydid).  
+The master branch of the source code can be cloned from the Github repository in the standard way (assuming you have Git installed and configured on your computer):
+
+    > git clone https://github.com/project8/katydid.git
+
+Follow the installation instructions in the top-level README file, which also shows up on the front page of the Github Katydid webpage. 
```

Comment:
Please provide a link to the README on github, where you mention it.

Reply:
link added in cff5c8d5